problems/2.functions.js

```diff
@@ -20,16 +21,22 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
-
+function incrementGlobalCount(val) {
+  
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val